src/views/bookmark_x/bookmark_tree_view.ts

```diff
@@ -110,4 +110,33 @@ export class BookmarkTreeViewManager {
             vscode.window.showInformationMessage("node is null");
         }
     }
-}
\ No newline at end of file
+
+    static async selectActiveGroup() {
+        let wsf = util.getWsfWithActiveEditor();
+        let cache = this.controller!.get_root_group(wsf!).cache;
+        let selectedFile: string | undefined = await vscode.window.showQuickPick(
+            (() => {
+                let options = ["root group"];
+                for (const element of cache.keys().slice(1)) {
+                    if (cache.get(element).type === "group") {
```

Comment:
you can use `typeIsGroupLike` function here to judge, because there is another type called `groupBookmark`
```typescript
import { typeIsGroupLike } from './constants';
if (typeIsGroupLike(cache.get(element).type)) { ... }
```